app/main.py

```diff
@@ -1,8 +1,27 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int):
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for person in people:
+        name = person["name"]
+        age = person["age"]
+        person_list.append(Person(name, age))
+
+    for person in people:
+        name = person["name"]
+        instance = Person.people[name]
+
+        if "wife" in person and person["wife"]:
+            instance.wife = Person.people[person["wife"]]
+        elif "husband" in person and person["husband"]:
+            instance.husband = Person.people[person["husband"]]
```

Comment:
The code attempts to assign `wife` and `husband` attributes to `Person` instances, but these attributes are not defined in the `Person` class. You should define these attributes in the `Person` class to avoid runtime errors.